Antidote/CallsManager.m

```diff
@@ -50,7 +50,7 @@ @implementation CallsManager
 - (instancetype)init
 {
     AALogVerbose();
-    self = [super self];
+    self = [super init];
```

Comment:
It's like the 3rd time I done that. Surprisingly not sure why it didn't cause any problems before.